src/components/feature.html

```diff
@@ -0,0 +1,26 @@
+<div class="main-feature">
+    <span>Features</span>
+    <h3>We Provide the Best Property for you</h3>
+    <div class="feature-img">
+        <div class="feature-imggroup">
```

Comment:
same. u should move it to component to reuse it. I see it being reused everywhere